src/components/carousel/Carousel.jsx

```diff
@@ -0,0 +1,76 @@
+import carousel from "./carousel.module.css";
+import { useState } from "react";
+import { v4 as uuid } from "uuid";
+
+function SliderLabels({ activeSlide, setActiveSlide, inputNum }) {
+  return (
+    <>
+      {inputNum.map((item) => {
+        return (
+          <label
+            key={uuid()}
+            htmlFor={item}
+            onClick={() => {
+              setActiveSlide(item);
+            }}
+            className={`${carousel["manual-btn"]}  
+              ${item === activeSlide ? carousel["active"] : "abcd"}`}
+          ></label>
+        );
+      })}
+    </>
+  );
+}
```

Comment:
Move this component in separate file

Reply:
this is a small component that is not used anywhere else throughout my application thats why i kept it in the same file for ease of understanding.